### ISSUE TITLE
Last import step on local development isn't working properly.

### ISSUE THREAD

**Issue by ashokdhaduk** [wptrainingteam/learn-theme-beta#59]
See this - http://prntscr.com/p7nx7a

**Comment by ashokdhaduk** [comment]
I found that there are some changes done after imported xml file. Like updated post type slugs. see this http://prntscr.com/p7o1r9

Import file has post type lesson_plan but admin view shows lesson-plan slug

**Comment by jessecowens** [comment]
@ashokdhaduk You're right, we need to update the XML file. In the meantime, use the WP-CLI commands in step 7 rather than the XML file
